frontend/packages/core/src/AppLayout/stories/logo.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
Just for my understanding, it seems like you need to have a `stories` dir in each component and thats automatically pulled in by storybook at build time?

Reply:
yupp, we can update this to be any pattern: https://github.com/lyft/clutch/pull/489/files#diff-c0daeb9d52f8778948a4e8a2835a953dR3